include/irr/core/Types.h

```diff
@@ -37,7 +37,9 @@ namespace core
 template<typename T>
 using allocator = _IRR_DEFAULT_ALLOCATOR_METATYPE<T>;
 
-
+//idk where to put this
```

Comment:
where it was, its a IBuffer only thing

Reply:
but i had to change namespace of IBuffer from core to asset

Reply:
either make an exception for IBuffer (needs to be in `core`) or create a intermediate class in the inheritance chain between `IGPUBuffer`,`ICPUBuffer` and `IBuffer`